Frontend/src/Components/BatteryCells/BatteryCells.js

```diff
@@ -1,147 +1,214 @@
-import { Box, Flex, Spacer, SimpleGrid, Heading, Text, VStack } from "@chakra-ui/react";
+import { SimpleGrid, VStack } from "@chakra-ui/react";
+import DataPack from "./DataPack";
 import CellGroup from "./CellGroup";
 
 export default function BatteryCells(props) {
     return (
-
-        <VStack>
+        <SimpleGrid
+            columns={2}
+            rows={1}
+            spacingX={'0.25vw'}
+            h="calc(100% - 26px)" // Calculate height of container (100%) - height of Select (24px) and border (2px)
+        >
+            <SimpleGrid
+                columns={1}
+                overflowY="scroll"
+            >
+                <DataPack
+                    dataTitle="Power Input Voltage"
+                    dataValue={props.data?.bms_input_voltage[0] ?? 5.0}

```

Comment:
```suggestion
                    dataValue={props.data?.bms_input_voltage[0] ?? 0.0}
```
Using 0 as a default rather than 5 (for both pack data and the cell groups) would make it more apparent that the actual value isn't available.

Reply:
Alternatively, would using a -1.0 be a better indicator that a value isn't available?

Reply:
Yes, go with -1